src/components/ProgressBar/index.tsx

```diff
@@ -19,7 +19,7 @@ const ProgressBarInternal = (props: ProgressBarProps) => {
     return () => {
       animation.stop();
     };
-  }, []);
+  }, [props.progress]);
```

Comment:
do we need props.progress in useEffect dependency ?

Reply:
made the requested changes.